tests/cli/test_export_cli.py

```diff
@@ -139,3 +139,11 @@ def test_dynamic_batching(self):
                 shell=True,
                 check=True,
             )
+
+    def test_stable_diffusion(self):
+        with tempfile.TemporaryDirectory() as tempdir:
+            subprocess.run(
+                f"optimum-cli export neuron --model hf-internal-testing/tiny-stable-diffusion-torch --task stable-diffusion --batch_size 1 --num_channels 4 --height 64 --width 64 --sequence_length 18 {tempdir}",
+                shell=True,
+                check=True,
```

Comment:
Why?

Reply:
Without `shell=True`, python will tru to find an executable whose file name is the entire string passing in.

```
E               FileNotFoundError: [Errno 2] No such file or directory: 'optimum-cli export neuron --model hf-internal-testing/tiny-stable-diffusion-torch --task stable-diffusion --batch_size 1 --num_channels 4 --height 64 --width 64 --sequence_length 18 /tmp/tmpqq_g71l9'
``` 

check as I need it to return the error when returning non-zero.

Reply:
You just need to split the command.

Reply:
It's true... just felt there are too many commas to add, but will do!